src/sqlfluff/rules/L057.py

```diff
@@ -54,56 +54,45 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
         self.unquoted_identifiers_policy: str
         self.allow_space_in_identifier: bool
 
-        if (
-            context.segment.name == "naked_identifier"
-            and identifiers_policy_applicable(
-                self.unquoted_identifiers_policy, context.parent_stack
-            )
-            and not (
-                context.segment.raw.replace("_", "").isalnum()
-                or (
-                    self.allow_space_in_identifier
-                    and context.segment.raw.replace("_", "").replace(" ", "").isalnum()
-                )
```

Comment:
This bit is completely redundant, you can't have spaces in an naked_identifier (n.b. that was from the original pr not yours)